HISTORY.rst

```diff
@@ -2,6 +2,15 @@
 History
 =======
 
+0.5.7-dev
+---------
+* Updated dependencies
+* Remove `bitcoin` import from `Crypto` exposing confusing functions
+* Remove `logzero` dependency and throw exceptions instead of logging them
+* Cleanup some docstrings and code
+* `ReadVarBytes` now uses `SafeReadBytes` internally
```

Comment:
I recommend adding an update here from https://github.com/CityOfZion/neo-python-core/pull/177. e.g.

`Crypto.Sign` now initializes the secp256r1 Elliptic curve parameters without initializing a `CryptoInstance` or using `KeyPair` prior.

Reply:
I intended to merge this one before #177 and then update the changelog on your PR when merging. For whatever reason this one decided to fail and I figured I'd continue with what I was doing to stay in the flow instead of trying to debug this. Will figure it out later